src/python/py_tokenizer.cpp

```diff
@@ -169,5 +169,19 @@ void init_tokenizer(py::module_& m) {
         .def("get_eos_token_id", &Tokenizer::get_eos_token_id)
         .def("get_pad_token", &Tokenizer::get_pad_token)
         .def("get_bos_token", &Tokenizer::get_bos_token)
-        .def("get_eos_token", &Tokenizer::get_eos_token);
+        .def("get_eos_token", &Tokenizer::get_eos_token)
+        .def("get_vocab",
```

Comment:
Please add a test to https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/test_tokenizer.py

Can you remind me the usecase? I'm wondering if we could add it to a sample so people new about this function.

Reply:
Tests are tricky in this case because our vocab differs from the original HF vocab.